config/initializers/rack_mini_profiler.rb

```diff
@@ -0,0 +1,7 @@
+if defined? Rack::MiniProfiler
+  # enable profiling for all pages; per-default it is disabled on production-env
+  Rack::MiniProfiler.config.authorization_mode = SETTINGS.fetch(:profiler_authorization_mode, :allow_all)
```

Comment:
In case it got lost, you can add the setting here in `config/settings.yaml.example`